civihr_employee_portal/civihr_employee_portal.install

```diff
@@ -317,6 +317,37 @@ function civihr_employee_portal_update_7020() {
   _civihr_employee_portal_create_default_reports_configuration();
 }
 
+/**
+ * Renames some roles. Basically the civihr_admin, civihr_staff
+ * civihr_manager and civihr_admin_local roles.
+ */
+function civihr_employee_portal_update_7021() {
+  $oldToNewNames = [
+    'civihr_admin' => 'HR Admin',
+    'civihr_staff' => 'Staff',
+    'civihr_manager' => 'Manager',
+    'civihr_admin_local' => 'Regional HR Admin'
+  ];
+
+  $roles = user_roles();
+
+  foreach ($roles as $id => $roleName) {
+    if (array_key_exists($roleName, $oldToNewNames)) {
+      $roleObject = user_role_load($id);
+      $roleObject->name = $oldToNewNames[$roleName];
+      user_role_save($roleObject);
+
+      //The role delegation module has dynamic permissions that changes
+      //when a role name is updated in UI via hook form_user_admin_role_submit
+      //We need to update it manually here.
+
+      db_update('role_permission')
```

Comment:
Wouldn't reverting the civihr_default_permissions feature have the same effect?

Reply:
It did have the same effect but I discovered that the previous permissions were left in the database causing redundant data.